README.md

```diff
@@ -307,28 +309,28 @@ all effects. Instead we can use `send` for unmatched cases.
 
 ```purescript
 -- This now interprets it back into `Run` but with the `EFF` effect.
-handleTalk :: forall eff r. TalkF ~> Run (eff :: EFF (console :: CONSOLE | eff) | r)
+handleTalk :: forall r. TalkF ~> Run (effect :: EFFECT | r)
 handleTalk = case _ of
   Speak str next -> do
-    -- `liftEff` lifts native `Eff` effects into `Run`.
-    liftEff $ Console.log str
+    -- `liftEffect` lifts native `Effect` effects into `Run`.
+    liftEffect $ Console.log str
     pure next
   Listen reply -> do
     pure (reply "I am Groot")
 
 runTalk
-  :: forall r eff
-   . Run (eff :: EFF (console :: CONSOLE | eff), talk :: TALK | r)
-  ~> Run (eff :: EFF (console :: CONSOLE | eff) | r)
+  :: forall r
+   . Run (effect :: EFFECT, talk :: TALK | r)
+  ~> Run (effect :: EFFECT | r)
 runTalk = interpret (on _talk handleTalk send)
 
-program2 :: forall eff r. Run (eff :: EFF (console :: CONSOLE | eff), dinner :: DINNER | r) Unit
+program2 :: forall r. Run (effect :: EFFECT, dinner :: DINNER | r) Unit
 program2 = dinnerTime # runTalk
 ```
 
-We've interpreted the `TALK` effect in terms of native `Eff` effects, and so
+We've interpreted the `TALK` effect in terms of native `Effect`'s, and so
```

Comment:
I would say "in terms of the native `Effect` type" to avoid the awkward `'s`.

Reply:
updated